pkg/k8s/k8s.go

```diff
@@ -79,7 +83,9 @@ func WaitForServiceAccount(cs *clients.Clients, ns, targetSA string) *corev1.Ser
 		}
 		return false, err
 	})
-	assert.NoError(err, fmt.Sprintf("ServiceAccount: %s, not found in namespace %s\n", targetSA, ns))
+	if err != nil {
+		testsuit.T.Errorf("service account %s not found in namespace %s \n %v", targetSA, ns)
```

Comment:
Do we need `\n` when we want to print just the namespace?

Reply:
good catch, I will fix this copy&paste error :)